charts/clearblade-iot-enterprise/charts/cb-postgres/templates/postgres-cluster.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if .Values.global.dynamicVolumes }}
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
```

Comment:
@WorkForPizza Have we ever tested pg clusters with the platform? Should we add that to the jenkins pipeline/local scripts?